sw/host/tests/manuf/provisioning/ft/src/main.rs

```diff
@@ -45,15 +45,15 @@ pub struct ManufFtProvisioningDataInput {
     host_ecc_sk: PathBuf,
 
     /// Measurement of the ROM_EXT image to be loaded onto the device.
-    #[arg(long, default_value = "0x00000000_00000000_00000000_00000000")]
+    #[arg(long, default_value = "0x11111111_11111111_11111111_11111111")]
```

Comment:
If these are sha256sums, then the sample values are half the length of a actual sha256 value.

Reply:
fixed.